packages/ra-core/src/controller/input/useReferenceInputController.ts

```diff
@@ -121,7 +121,7 @@ export const useReferenceInputController = <RecordType extends RaRecord = any>(
             enabled: currentValue != null && currentValue !== '',
             meta,
             ...otherQueryOptions,
-        },
+        } as UseQueryOptions<RecordType[], Error>,
```

Comment:
If this is supposed to fix the TS type, can you please remove the `@ts-ignore` at line 119 which should no longer be necessary?
Thanks.

Reply:
When I say fail to build. i was getting errors when building my project with npm run build. However I re-installed everything from the beginning and a new .eslintrc file was created and it worked this time. However these issues are going red in the IDE when using typescript